RELEASE_NOTES.md

```diff
@@ -9,6 +9,227 @@ incompatible changes have been made in that release.
 For a list of all the changes in a particular release, please refer to the changelog file -
 https://github.com/scalyr/scalyr-agent-2/blob/master/CHANGELOG.md.
 
+## 2.1.34 "Tadongolia" - August 30, 2022
```

Comment:
You can remove those changes from this file - we use release notes file to provide additional context and information on larger / breaking changes, but there is no need to repeat stuff from changelog file.